### PR TITLE
Release 3.2.0

### PR DESCRIPTION
**Pull request by ggabernet**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/airrflow/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/airrflow _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,37 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [3.1] - 2023-06-05 "Protego"
+## [3.2.0dev] -
```
Comment (ggabernet) [inline CHANGELOG.md:6]:
```suggestion
## [3.2.0] - 2023-10-23 Expecto patronum
```

**Comment by ggabernet** [comment]
The `fetchimgt` tests are failing at the moment, pulling the data from IMGT is not so stable but the pipeline uses the cached databases by default. Re-running these tests tomorrow should make them pass by experience...

File: assets/schema_input.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        },
+        "required": [
+            "sample_id",
+            "subject_id",
+            "species",
+            "pcr_target_locus",
+            "tissue",
+            "sex",
+            "age",
+            "biomaterial_provider"
```
Comment (ssnn-airr) [inline assets/schema_input.json:60]: Is `single_cell` missing in `required`? The error message is "Single cell must be provided..."

Comment (ggabernet) [reply assets/schema_input.json:60]: yes, you're right! I'll add it now :)

Comment (ggabernet) [inline assets/schema_input.json:60]:
```suggestion
            "biomaterial_provider",
            "single_cell"
```

**Review by ssnn-airr — APPROVED** [review]

**Review by pinin4fjords — APPROVED** [review]
Deferring somewhat to individual PR reviewers, but looks good to me.

File: CHANGELOG.md [hunk 3]
```diff
@@ -206,6 +236,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - [#104](https://github.com/nf-core/bcellmagic/pull/104): Fix bug in pairseq barcode copy before consensus.
 - [#114](https://github.com/nf-core/bcellmagic/pull/114): Analysis not restricted to Ig heavy chains.
 - [#123](https://github.com/nf-core/bcellmagic/pull/123): Fix report Rmarkdown reading for running on AWS.
+- [#284](https://github.com/nf-core/bcellmagic/pull/284): Use cached IMGT and IgBlast reference data by default.
```
Comment (ggabernet) [inline CHANGELOG.md:239]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,7 +3,37 @@
...
+- [#276](https://github.com/nf-core/airrflow/pull/276) Parametrize FindThreshold Report and Presto Buildconsensus UMI.
+- [#281](https://github.com/nf-core/airrflow/pull/281) Update to nf-core tools v2.10.
+
+### `Fixed`
+
+- [#268](https://github.com/nf-core/airrflow/pull/268) Allows for uppercase and lowercase locus in samplesheet `pcr_target_locus`.
+- [#259](https://github.com/nf-core/airrflow/pull/259) Samplesheet only allows data from one species.
+- [#259](https://github.com/nf-core/airrflow/pull/259) Introduced fix for a too long command with hundreds of datasets.
+- [#266](https://github.com/nf-core/airrflow/pull/266) Convert samplesheet required columns to strings when needed.
+- [#276](https://github.com/nf-core/airrflow/pull/266) Temporary fix for Dowser with singularity
```
Comment (ggabernet) [inline CHANGELOG.md:24]:
```suggestion
- [#284](https://github.com/nf-core/bcellmagic/pull/284): Use cached IMGT and IgBlast reference data by default.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,7 +3,37 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [3.1] - 2023-06-05 "Protego"
+## [3.2.0] - 2023-10-24 Expecto patronum
```
Comment (ggabernet) [inline CHANGELOG.md:6]:
```suggestion
## [3.2.0] - 2023-10-27 Expecto patronum
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,7 +3,37 @@
...
+- [#259](https://github.com/nf-core/airrflow/pull/259) Samplesheet only allows data from one species.
+- [#259](https://github.com/nf-core/airrflow/pull/259) Introduced fix for a too long command with hundreds of datasets.
+- [#266](https://github.com/nf-core/airrflow/pull/266) Convert samplesheet required columns to strings when needed.
+- [#276](https://github.com/nf-core/airrflow/pull/266) Temporary fix for Dowser with singularity
+
+### `Dependencies`
+
+| Dependency | Old version | New version |
+| ---------- | ----------- | ----------- |
+| r-enchantr | 0.1.2       | 0.1.7       |
```
Comment (ggabernet) [inline CHANGELOG.md:30]:
```suggestion
| r-enchantr | 0.1.2       | 0.1.9       |
```

**Pull request merged**